packages/server/lib/screenshots.ts

```diff
@@ -316,6 +333,7 @@ const ensureSafePath = function (withoutExt, extension, overwrite, num = 0) {
     }
 
     // path does not exist, attempt to create it to check for an ENAMETOOLONG error
+    // @ts-ignore
     return fs.outputFileAsync(fullPath, '')
```

Comment:
I think this might fix the ts-ignore

```suggestion
    return fs.outputFile(fullPath, '')
```

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)